### ISSUE TITLE
Fullpage.js multi instances and disable scroll with add/removeClass

### ISSUE THREAD

**Issue by artndesignfactory** [alvarotrigo/fullPage.js#1959]
Hello,

First of all I have to thank everyone involved in making this amazing piece of work! Now to my issues...

I am trying to build my online portfolio, which in itself is a long battle and struggle. jQuery is not my strong side, which is why I came to you for guidance. I have stumbled upon some issues, in which I would like to get some clarification and I hope you will at some point find the time to reply. :)

**My first issue is:**
I run fullpage on my homepage and have a few cases displayed within it, and it looks great. I have added a little jQuery in order to create a lightbox/popup div, and within it I display my cases. And this is where issue 1 comes in.

In the case div it is scrollable, which means the fullpage div is running in the background and scrolling while the case div is open. I would love to know how and if you could disable the scrolling within fullpage by simply add/removeClass somehow? That could potentially fix my issue 1. Would be the perfect solution if a class could be added to disable/enable the scroll.

**Second issue**
I have tried to research quite a bit into this issue. I would love to run fullpage twice on my website (onepage website). I have also looked into alternatives where I replace one of the instances with another carousel or other solutions, but I would prefer to use fullpage.js only, because I like the entire functionality much better compared to others I have researched (which is A LOT).

1 x On the homepage.
1 x On the homepage, but within lightbox/popup div. (Which displays on click).

So basically on the same page, just one is displayed at the very beginning and one is hidden within a div, which becomes visible like a lightbox. I can easily get one to work, either on landing or within the lightbox div. The issue I have, is that, if I run both at the same time, it totally bugs up.

I have read several posts and topics asking this question, on other forums also, but it does not seem like anyone has been able to find a solution for it. I know the script is built for one instance, but it seems like I am not alone in wanting to run it multiple times.

_I hope you will be able and find the time to answer my two questions. Once again amazing piece of work you have done!_

Best regards, Morten.

**Comment by alvarotrigo** [comment]
> I would love to know how and if you could disable the scrolling within fullpage by simply add/removeClass somehow?

Use the option `normalScrollElements` or if you prefer, the function `setAllowScrolling(false)` when opening the dialog.

> 1 x On the homepage, but within lightbox/popup div. (Which displays on click).

You can try it by making use of an iframe. But fullpage.js doesn't provide any option to create multiple instances of it at the moment, and less yet one over another.
